This file is part of MXE. See LICENSE.md for licensing information.

Taken from https://github.com/tpaviot/oce/pull/673

--- a/src/BRepBuilderAPI/BRepBuilderAPI_Copy.cxx
+++ b/src/BRepBuilderAPI/BRepBuilderAPI_Copy.cxx
@@ -25,8 +25,6 @@
 #include <gp_Pnt.hxx>
 #include <Poly_Triangulation.hxx>
 
-namespace {
-
 //! Tool class implementing necessary functionality for copying geometry
 class BRepBuilderAPI_Copy_Modification : public BRepTools_Modification 
 {
@@ -147,8 +145,6 @@ DEFINE_STANDARD_HANDLE(BRepBuilderAPI_Copy_Modification, BRepTools_Modification)
 IMPLEMENT_STANDARD_HANDLE(BRepBuilderAPI_Copy_Modification, BRepTools_Modification)
 IMPLEMENT_STANDARD_RTTIEXT(BRepBuilderAPI_Copy_Modification, BRepTools_Modification)
 
-} // anonymous namespace
-
 //=======================================================================
 //function : BRepBuilderAPI_Copy
 //purpose  : 
